index.template.html

```diff
@@ -5518,28 +5518,102 @@ <h1>JSON-LD Context Usage</h1>
+  <h2 id="changes-from-fpwd">Changes from First Public Working Draft 24 November 2020</h2>
     <ul>
-      <li>Section <a href="#validation-serialization-json"></a> was added to define multiple levels of
-          validation, appropriate for different use cases including development, deployment,
-          and directory registration.</li>
-      <li>Section <a href="#canonicalization-serialization-json"></a> was added to define a 
-          canonical serialization, useful for signing and comparison of Thing Descriptions.</li>
-      <li>Section <a href="#sec-data-schema-vocabulary-definition"></a> defines a list of content types that can be used with data schema.</li>
-      <li>The additional term <em>additionalResponses</em> was added to section <a href="#form"></a>.  These can be used to define error responses.</li>
-      <li>Section <a href="#sec-data-schema-vocabulary-definition"></a> defines a list of content types that can be used with data schema.</li>
-      <li>Additional terms <em>contentEncoding</em> and <em>contentMediaType</em> were added in section <a href="#dataschema"></a>.  There is an <a href="#example-26">Example 26</a> that shows how to use them to describe binary data embedded in text-based values.</li>
-      <li>An additional term <em>multipleOf</em> was added in section <a href="#numberschema"></a>.</li>
-      <li>Additional terms <em>minLength</em>, <em>maxLength</em> were added in section <a href="#stringschema"></a>.</li>
-      <li>A new security scheme <a href="#combosecurityscheme"></a> was added.</li>
-      <li>Section <a href="#oauth2securityscheme"></a> was updated with two additional flows <em>client</em> and <em>device</em>. The section also notes that all other flows besides the ones defined in the section should be specified using an extension.</li>
-      <li>In section <a href="#x5-4-default-value-definitions"></a>, the default value definition given for <em>op</em> term in <em>EventAffordance</em>'s <em>form</em>s was changed.</li>
-      <li><a href="#example-23">Example 23</a> was updated to show an example use of URI Template variables in a property definition.</li>
-      <li>Section <a href="#semantic-annotations-example-geoloc"></a> was added to describe three examples that show how to embed location based data or annotate data schema elements with location based semantics.
-</li>
-      <li>The section describing <a href="#thing-model"></a> was moved to the main body of this document. It was previously in the appendix.</li>
+      <!-- 3 -->
+      <li>In section <a href="#terminology"></a>, a new WoT terminology <a>Canonical TD</a> was given definition.</li>
+      <!-- 4 -->
+      <li>In section <a href="#namespaces"></a>, Thing Model vocabulary was assigned a namespace IRI.</li>
+      <!-- 5.3.1.1 -->
+      <li>In section <a href="#thing"></a>:
+        <ul>
+          <li>New terms <code>base</code>, <code>securityDefinitions</code> and <code>profile</code> were added.</li>
```

Comment:
I think, this can be removed. Originally, these were already included. It is unclear why those terms dissipated in the last update.